docs/user-docs/latest/content-manager/translating-content.md

```diff
@@ -14,7 +14,9 @@ In the Content Manager, when the Internationalization plugin is installed, some
 - an Internationalization box, displayed in the right side of the interface, from where it is possible to switch locales
 - icons displayed next to every field to indicate whether the field can be translated or not:
   - <img width="16" src="../assets/content-manager/icon_localizable.png"> indicates that the field can be translated,
-  - <img width="18" src="../assets/content-manager/icon_non-localizable.png"> indicates that the field cannot be translated: its content is the same for every locale.
+  - <img width="18" src="../assets/content-manager/icon_non-localizable.png"> indicates that the field cannot be translated: its content is the same for every locale (i.e. changing the value of a non-localizable field changes it for all other locales).
+
+The Internationalization plugin also allows components and dynamic zones to have a different structure from one locale to another. In other words: the order of the fields and components of an entry, as well as the number of times repeatable components are used, do not have to be the same for all locales.
```

Comment:
Hello, I'm not sure about the meaning of this sentence.

DZ can have a different structure, that makes sense. You define a DZ and the data you add inside (components) could be different.

eg. DZ allow 3 compos - image, text, content
DZ en can have an image and text
DZ fr can have image and content

But saying that components can have a different structure, I don't really understand.
If a content type has a compo in its structure, it will be the same compo structure whatever the local.
And if we use it as repeatable, the number of repetitions can be different, and here it makes sense.

Happy to discuss this comment.